pkg/scheduler/schedule_one_test.go

```diff
@@ -1204,6 +1213,202 @@ func TestSchedulerScheduleOne(t *testing.T) {
 	}
 }
 
+func TestSignatures(t *testing.T) {
+	testPod := podWithID("foo", "")
+	testPodLabel := podWithID("label", "")
+	testPodLabel.Labels = map[string]string{"foo": "bar"}
+	testPodPort := podWithPort("bar", "", 15)
+	testPodAffinity := podWithAffinity("bat", "", "foo")
+	testPodPortRes := podWithPort("bar", "", 16)
+
+	cpu := int64(100)
+	mem := int64(400000)
+	testPodRes := podWithResources("baz", "", v1.ResourceList{
+		v1.ResourceCPU:    *(resource.NewQuantity(cpu, resource.DecimalSI)),
+		v1.ResourceMemory: *(resource.NewQuantity(mem, resource.DecimalSI)),
+	}, v1.ResourceList{
+		v1.ResourceCPU:    *(resource.NewQuantity(cpu, resource.DecimalSI)),
+		v1.ResourceMemory: *(resource.NewQuantity(mem, resource.DecimalSI)),
+	})
+
+	testPodPortRes.Spec.Containers[0].Resources.Requests = v1.ResourceList{
+		v1.ResourceCPU:    *(resource.NewQuantity(cpu, resource.DecimalSI)),
+		v1.ResourceMemory: *(resource.NewQuantity(mem, resource.DecimalSI)),
+	}
+
+	podSpreadNoDefaults := &schedulerapi.PodTopologySpreadArgs{DefaultingType: schedulerapi.ListDefaulting}
+	nodeAff := &schedulerapi.NodeAffinityArgs{}
+	volBind := &schedulerapi.VolumeBindingArgs{}
+
+	fts := feature.Features{}
+
+	table := []struct {
+		name                 string
+		sendPod              *v1.Pod
+		registerPluginFuncs  []tf.RegisterPluginFunc
+		expectedSignature    string
```

Comment:
It's very annoying to maintain JSON-marshalled signature in each test case. 
Is it somehow possible to change it to some structured one?